### ISSUE TITLE
Havok DH - Disable Immolation Aura always recommending back to back

### ISSUE THREAD

**Issue by gonzoinc** [Hekili/hekili#3208]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this issue (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Describe the Issue

During rotation Immolation aura is always being recommended back to back, this is not optimal and it is causing fury resource generation issues as this is the primary fury generator there is a lot of downtime waiting for this to come off cooldown or for fury to regenerate through Demon Blades. Siming does not show this ever being used back to back and I see no option to disable this.

[Havok_DH_Snapshot.txt](https://github.com/Hekili/hekili/files/14926352/Havok_DH_Snapshot.txt)
![Havok_DH_Snapshot](https://github.com/Hekili/hekili/assets/27728728/0ba90907-fe42-4f10-810f-8ccff26add99)

### How to Reproduce

1. Target  Dummy
2. Immolation Aura is shown back to back

### Snapshot (Link)

https://pastebin.com/e0b0DJFB

### Raidbots Sim Report (Link)

https://www.raidbots.com/simbot/report/eLxbrdCRXvpwX96rzdXP5n

**Comment by Hekili** [comment]
This is because the addon doesn't assume combat has started when you press Immolation Aura (i.e., you can press it and hit 0 enemies around you).  When you pop the first Immolation Aura, if you're actually in range of enemies, the next recommendation will change.

**Comment by gonzoinc** [comment]
It is still recommending them back to back, I just can't capture a snapshot of it as it will change as soon as I try. Dropping the queue down from 4 to 3 abilities seemed to help a little but every now and then the double immolation aura popped back in.  I've also found that it's not recommending it when the aura is not up and there are charges. I've just disabled the ability in Hekili and configured a weak aura to track the aura to ensure I can keep it up and to recommend the second charge if Fury gets below X. Thx!
